StratisCore.UI/src/app/shared/components/side-bar/side-bar.component.ts

```diff
@@ -38,7 +38,8 @@ private modalService: NgbModal, private globalService: GlobalService, private cu
     }, {
       selected: false,
       displayText: 'Staking',
-      visible: of(true),
+      visible: of(!this.globalService.getSidechainEnabled()),
```

Comment:
This will be conflicted now, as SideBarItems are Dependency Injected to allow for more complex implementations. 
There's a StakingSidebarItem that effectively does this + shows different class when staking

Reply:
Ok, I will resolve the conflict when merging